clang/lib/Sema/SemaDecl.cpp

```diff
@@ -7353,6 +7353,15 @@ void emitReadOnlyPlacementAttrWarning(Sema &S, const VarDecl *VD) {
   }
 }
 
+// Checks if VD is declared at global scope or with C language linkage
```

Comment:
```suggestion
// Checks if VD is declared at global scope or with C language linkage.
```
Extremely minor nit.

Reply:
Fixed